modular-docs-manual/content/topics/whats-new.adoc

```diff
@@ -0,0 +1,18 @@
+[id="whats-new_{context}"]
+= What's new
+
+The following list summarizes significant changes to the mod docs standard over time.
+In most cases, apply these changes to your new content and revisions.
```

Comment:
I'm not sure what you're trying to say - with the next line I think you're saying you don't have to change anything in existing content, just new. Which means we don't need this line at all.

Reply:
Thanks. I hope the updated version is clearer.